tools/cmdline/src/test/test.js

```diff
@@ -14,29 +14,28 @@ describe('cmdline-exec-utility', () => {
     sandbox.restore();
   });
 
-  it('executes command that writes in stdout' + 
+  it('executes command that writes in stdout' +
   ' (command is included in stdout)', () => {
-    let result = exec('ls -l');
+    const result = exec('ls -l');
     expect(result !== null);
     expect(console.log.calledOnce).to.be.equal(true);
   });
 
   it('executes command that writes in stdout' +
   ' (command is not included in stdout)', () => {
-    let result = exec('ls -l', false);
+    const result = exec('ls -l', false);
     expect(result !== null);
     expect(console.log.calledOnce).to.be.equal(false);
   });
 
-  it('executes command that writes in stderr' + 
+  it('executes command that writes in stderr' +
   ' (command is included in stdout)', () => {
     try {
```

Comment:
Better replace this with http://chaijs.com/api/bdd/#method_throw
Will read as `expect(() => exec('ls foo')).to.throw()`